packages/create-pantheon-decoupled-kit/src/utils/runESLint.ts

```diff
@@ -25,9 +25,14 @@ export const runESLint = (
 		execSync(`${command} lint`, { cwd: answers.outDir, stdio: 'inherit' });
 		return 'success';
 	} catch (error) {
-		if (error instanceof Error) {
-			throw error;
+		try {
+			execSync('npx eslint', { cwd: answers.outDir, stdio: 'inherit' });
+			return 'success eslint';
+		} catch (error) {
+			if (error instanceof Error) {
+				throw error;
+			}
+			throw 'fail';
```

Comment:
Not a huge fan of this pattern as it will still show some nasty errors in the console. My suggestion would be to read from the `scripts` of the new`package.json` to see if there is a `lint` script, then call that script if it exists, otherwise use `npx eslint --write ` with configurable options like ignore pattern/path or plugins. Meaning, I should be able to pass options ito the `runESLint` config that are then passed into this command.